internal/layout/box.go

```diff
@@ -10,6 +10,7 @@ var _ fyne.Layout = (*Box)(nil)
 // Box is a box layout for stacking a number of child canvas objects horizontally or vertically.
 type Box struct {
 	Horizontal        bool
+	PadBeforeAndAfter bool
```

Comment:
Learned a new word today :).
But it is not the right in this context. Interstitial padding is always added. This flag is for additional padding before or after the objects.

Reply:
Can you expand on why this is needed please? If we were using the old popup style menus the padding would already be provided but that was removed.

Reply:
Oh wow I only just realised that this used to be layout/Box but now is internal/layout/Box.

I’m getting quite nervous about the number of things we are pulling into internal so that we can make changes that otherwise would add public APIs.

Reply:
I reverted the layout change.